domains/_vercel.gm.json

```
@@ -0,0 +1,4 @@
+{
+  "owner": { "username": "donotfeedthevirus", "email": "[email protected]" },
```

you can safely delete this if you dont want to share your email 
```suggestion
  "owner": { "username": "donotfeedthevirus" },
```